gateway/1.5/modules/ROOT/pages/flex-local-configuration-reference-guide.adoc

```diff
@@ -484,29 +499,97 @@ spec:
 |`sharedStorage.redis.address`
 |Required
 |N/A
-|The redis instance address in `host:port` format.
+|The Redis instance address in `host:port` format
 
 |`sharedStorage.redis.username`
 |Optional
 |Empty
-|The redis user name.
+|The Redis user name
 
 |`sharedStorage.redis.password`
 |Optional
 |Empty
-|The redis user password.
+|The Redis user password
 
 |`sharedStorage.redis.db`
 |Optional 
 |0
-|The redis database number to use.
+|The Redis database number to use
+
+|`sharedStorage.redis.tls`
+|Optional
+|Empty
+|The Redis TLS configuration
+
+| `sharedStorage.redis.tls.trustedCA`
+| Required
+| N/A
+| The client certificate used in TLS
+
+| `sharedStorage.redis.tls.certificate`
+| Required
+| N/A
+| A TLS certificate key pair
+
+| `sharedStorage.redis.tls.keyPassphrase`
+| Optional
+| `none`
+| The certificate key passphrase. If specified, use the private key format that includes a DEK-Info header. If unspecified, the private key is not encrypted.
```

Comment:
The customer could still provide an encrypted private key, but it will fail if the passphrase is missing

```suggestion
| The certificate key passphrase. If specified, use the private key format that includes a DEK-Info header. If unspecified, the private key must not be encrypted.
```